jenkins/jobs/vic-scenario/build.sh

```diff
@@ -125,7 +125,7 @@ pushd vic
         echo "Tarball extraction passed, Running nightlies test.."
     fi
 
-    pabot --processes ${PARALLEL_JOBS} --removekeywords TAG:secret ${excludes} --variable ESX_VERSION:"${ESX_BUILD}" --variable VC_VERSION:"${VC_BUILD}" -d report "${testcases[@]}"
+    pabot --processes ${PARALLEL_JOBS} ${excludes} --variable ESX_VERSION:"${ESX_BUILD}" --variable VC_VERSION:"${VC_BUILD}" -d report "${testcases[@]}"
```

Comment:
I don't think we want to make this change; removing secret information from logs is important.

Reply:
OK. I will revert it back, but it is really difficulty for troubleshooting without logging them.